components/emoji/add_emoji/add_emoji.tsx

```diff
@@ -291,11 +382,9 @@ export default class AddEmoji extends React.PureComponent {
                                 />
                             </Link>
                             <SpinnerButton
-                                className='btn btn-primary'
-                                type='submit'
```

Comment:
I was avoiding changing the SpinnerButton since it touches a lot of areas of the app I'm not familiar with.  If there's end-to-end coverage and you have a chance to confirm manually on the test server, I'd feel safe that I didn't regress anything.

All the implementations specify `btn btn-primary` and `type='submit'` so we shouldn't need default values.

- add_emoji
- abstract_command
- abstract_incoming_webhook
- abstract_oauth_app
- abstract_outgoing_webhook
- add_bot
- interactive_dialog

Reply:
I tried out Bot, Webhook, and Slash Command creation and they seem to work well.

Reply:
Yeah, that's fine to leave them as-is. Repeating ourselves with the props isn't much of an issue, and we can always go and change in the future if we feel the need to.